### ISSUE TITLE
I broke it

### ISSUE THREAD

**Issue by natakata** [EmmittJ/PoESkillTree#282]
https://www.google.com/url?q=http://poeurl.com/xxm&sa=D&ust=1457214529782000&usg=AFQjCNFQO2r5_DD-lsdcLY7CiIhlhEG_IQ

I copied the above link into the "skill tree link" box and it broke it by getting into a redirect loop where it is constantly "resolving shortened tree address.". all the text in the attributes boxes are flashing from activity. the url i posted changed to "http://poeurl.com/home.php?url=redirect.php?url=redirect.php?url=redirect.php?url=redirect.php?url=redirect.php?url=redirect.php?url=redirect.php?url=redirect.php?url=redirect.php?url=redirect.php"...........repeats a TON and ends with ...."?url=redirect.php?url=xxm

I was using a version i downloaded prior to the release of Ascendancy when this started and i updated to  what shows in the program info as 2.2.5

Steps i tried to fix it.

1 double clicked to load a saved build. did not work.
2 closed program and restarted it. It has the bad url in the box already and as soon as i click on it to past a new url it tries to load the bad url and does not allow me to paste in a new url or delete the old one.
3 updated program. did not work.
4 tried to find a file for the program that maybe had the the link stored somewhere so i could try altering that file to stop it. I failed to find it anywhere i could think to look or was avle to check with <ctrl>+F in notepad.

i hope this is useful to you in resolving the problem or preventing it in the future. It would be nice to know if there is a way i can clear this up without losing my saved builds.

Thanks for the otherwise wonderful tool.

**Comment by EmmittJ** [comment]
I'm alright with fixing stuff like this, but is there a reason why you aren't just copying the poeurl link?

**Comment by EmmittJ** [comment]
I also don't seem to have a problem with the link aboce

**Comment by EmmittJ** [comment]
1) [Find your PersistentData.xml file](https://github.com/EmmittJ/PoESkillTree/wiki/How-To:-Copy-Builds)
2) Open PersistentData.xml in NotePade
3) Find `<CurrentBuild>...<Url>...</Url>...</CurrentBuild>`
4) Replace everything between `<Url>` and `</Url>` with nothing.

**Comment by natakata** [comment]
Thanks EmmittJ! 

That is exactly what i was looking for and could not find.   Again, stellar tool.

**Comment by natakata** [comment]
EmmittJ,
I realized i never answered your question as to why i was not using the poeurl link. I was trying to research a build and came across this google document from twitchtv user projectpt that lists a bunch of builds he has been going over/putting together for the last week 

https://docs.google.com/spreadsheets/d/1BERc6ffsWECBxoNy4r5DPrzUpmubbdmzwu7LuZqAvts/htmlview?usp=sharing&sle=true

I tried to open one of the build links buy just clicking on it and got an error when the poe site tried to load it. I then followed the directions he posted in huge letters at the top of the page and tried it in your tool which i had already been using. That is what broke it. 

Thanks again.
